view/adminhtml/web/js/suggested_addresses_admin.js

```diff
@@ -124,7 +124,10 @@ define([
 
             button.unbind('click').bind('click', function (e) {
                 e.preventDefault();
-                self.validateAddress($(this));
+
+                if (!button.is(':hidden')) {
```

Comment:
@ryanreeves-taxjar Can you opt for `button.is(':visible')` instead so we can drop the `!` and make it easier to read? Thank you!